src/OpenTelemetry/Trace/Internal/NoopActivityProcessor.cs

```diff
@@ -22,30 +22,5 @@ namespace OpenTelemetry.Trace.Internal
 {
     internal sealed class NoopActivityProcessor : ActivityProcessor
```

Comment:
@cijothomas do we still need this or we can change the base class to a non-abstract one?

Reply:
Personal take: having an `NoopActivityProcessor` seems better as it is good to be explicit.